docs/user-guide/system-admin/configuration/installation.md

```diff
@@ -38,6 +39,10 @@ You'll find zip files for each platform, for example: `zanata-<version>-eap-6.zi
 
 You will also find `war` archives. Don't download these unless you wish to manually configure JBoss, or if you are upgrading an already set up Zanata server.
 
+To run Zanata with Docker, follow [these instructions](https://github.com/zanata/zanata-server/tree/master/docker). You will need this Docker image:
+
+- [Latest Zanata Docker image](https://hub.docker.com/r/zanata/server/)
```

Comment:
Thanks Sean


On Mon, Sep 12, 2016 at 10:29 AM, Sean Flanigan <notifications@github.com>
wrote:

> In docs/user-guide/system-admin/configuration/installation.md
> <https://github.com/zanata/zanata-server/pull/1288#discussion_r78306041>:
>
> > @@ -38,6 +39,10 @@ You'll find zip files for each platform, for example: `zanata-<version>-eap-6.zi
> >
> >  You will also find `war` archives. Don't download these unless you wish to manually configure JBoss, or if you are upgrading an already set up Zanata server.
> >
> > +To run Zanata with Docker, follow [these instructions](https://github.com/zanata/zanata-server/tree/master/docker). You will need this Docker image:
> > +
> > +- [Latest Zanata Docker image](https://hub.docker.com/r/zanata/server/)
>
> Those instructions (for devs only) don't go with the Docker Hub image. You
> might want these instructions: https://github.com/zanata/
> docker-images/blob/master/zanata-server/README.md
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/zanata/zanata-server/pull/1288/files/97c023c7514d90e3c800ef77454d97148342b7ea#r78306041>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ARVlSYSDUZUQYSB0NLvTgeRcDHuZQpeaks5qpJzXgaJpZM4J6JS0>
> .
>



-- 
[image: photo]
Kathryn Gough
Web Developer
Globalisation
Red Hat, Inc

61 7 3514  7119 // au.redhat.com // kgough@redhat.com // Brisbane, AU